core-web/apps/dotcms-ui/src/app/view/components/dot-toolbar/components/dot-toolbar-user/dot-toolbar-user.component.scss

```diff
@@ -27,3 +27,25 @@ p-avatar {
         color: $black;
     }
 }
+
+::ng-deep {
```

Comment:
Move this into the `:host` selector so it get encapsulated to only this component because this `:ng-deep` will affect all the `.p-menu` in the page and because this component lives in all pages we can get this filter out to other `p-menu` on the page.